src/types.ts

```diff
@@ -10,17 +10,12 @@ export interface AutoIncrementOptionsSimple {
 
 export type AutoIncrementSimplePluginOptions = AutoIncrementOptionsSimple | AutoIncrementOptionsSimple[];
 
-export interface AutoIncrementOptionsID {
+export interface AutoIncrementIDOptions {
   /**
    * How much to increment the field by
    * @default 1
    */
   incrementBy?: number;
-  /**
-   * The Tracker Collection to use to keep track of an counter for the ID
-   * @default idtracker
-   */
-  trackerCollection?: string;
```

Comment:
why remove this option?

Reply:
we let mongoose to apply their way of naming the collections. they will lowercase and pluralize the collection name from model name.

Reply:
but why not give the user the option to change it to something that suits to the style of naming things, including disabling pluralization

Reply:
ok will revert this change.